tools/pytorch2onnx.py

```diff
@@ -81,7 +81,7 @@ def pytorch2onnx(model,
 
         # check the numerical value
         # get pytorch output
-        pytorch_result = model(input_tensor)[0].detach().numpy()
+        pytorch_result = model(input_tensor).detach().numpy()
```

Comment:
For now we need this `[0]` because `dummy_forward` return a tuple.

https://github.com/open-mmlab/mmaction2/blob/c4978a54d0bfdd5a19cb7c64981fcba7350cf78e/mmaction/models/recognizers/recognizer2d.py#L163

But I don't understand why we return a tuple.

Reply:
I have no idea why it is necessary, seems that this `forward_dummy` is only used in flops calculation and model deployment. Any comments? @dreamerlin

Reply:
Since later we may have a universal tool for all repos, keep consistency with other repos and within this repo is important.

Reply:
It will be not equal with `onnx_result` if we remove `[0]`. BTW, `model(input_tensor)` will call `forward` rather than `forward_dummy`

Reply:
@dreamerlin In pytorch2onnx.py, we replace `forward` with `forward_dummy`, so `model(input_tensor)` will call `forward_dummy`

https://github.com/open-mmlab/mmaction2/blob/9ee7129e6ad86131154d00a0e809f082c801441b/tools/pytorch2onnx.py#L149-L152

Reply:
Ops, my fault.